frontend-nextjs/src/features/dashboard/fileupload/Button.tsx

```
@@ -6,18 +7,23 @@ import type React from "react";
 	children = "Upload Files",
+	disabled = false,
 }: ButtonWithIconProps) {
 	return (
 		<div className="flex items-center gap-4">
 			<Button
 				variant="gradient"
 				className="flex items-center gap-3"
 				onClick={onClick}
+				disabled={disabled}
+				aria-label={typeof children === "string" ? children : "Upload Files"}
+				role="button"
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)